src/Serilog/Core/Logger.cs

```diff
@@ -1372,5 +1373,21 @@ public void Dispose()
         {
             _dispose?.Invoke();
         }
+
+        /// <summary>
+        /// Backing property for the exposed None property
+        /// </summary>
+        static readonly ILogger _noneLogger = new SilentLogger();
+
+        /// <summary>
+        /// An <see cref="ILogger"/> instance that efficiently ignores all method calls
+        /// </summary>
+        public static ILogger None
```

Comment:
We could get away without a backing property by declaring this as:

```csharp
public static ILogger None { get; } = new SilentLogger();
```

Reply:
No worries - wasn't sure if you were happy with newer syntax